src/clj/clojurians_log/components/parser.clj

```diff
@@ -0,0 +1,23 @@
+(ns clojurians-log.components.parser
+  (:require [clojure.string :as str]))
+
+(defn- extract-texts
+  [messages]
+  (let [texts (map :message/text messages)]
+    (clojure.string/join " " texts)))
```

Comment:
you're already aliasing `clojure.string` to `str`, so might as well make this `(str/join ,,,)`